cpp2python.py

```diff
@@ -45,6 +45,23 @@ def is_source(filename):
     return False
 
 def process_line(line):
+    #DO WHILE
+    # regex = r"\s(do)\s*\{((\w|\W)*)\}\s*(while)?\(([a-zA-Z0-9\!\=\& ]+)\);"
+
+    # subst = "\\2\\4(\\5):\\n\\2"
+
+    # line = re.sub(regex, subst, line, 0, re.VERBOSE | re.MULTILINE)
+    
+    """ remove pointer
+
+        double *d
+          V
+        d
+    """
+    regex = r"(double|int|char|float)\s+\*\s?(\w+)\b"
```

Comment:
You replace only 4 standard types, but there are much more. And user defined types might exist. Is it possible to write more generic reg exp but still avoid false positive matches?

`\s` is declared mandatory after type name, but `int* i` is very common coding style.

I think variable name can include digits and underscores. Just make sure digit is not the first letter.